Frames.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Frames"
-  s.version      = "3.3.0"
+  s.version      = "3.4.0"
```

Comment:
need to also make this change in `Package.swift` and `CheckoutApiClient+Constants.swift`